test/functional/discovered_hosts_controller_test.rb

```diff
@@ -285,7 +285,7 @@ def test_multiple_reboot_error
     ::ForemanDiscovery::NodeAPI::PowerService.any_instance.expects(:reboot).raises("request failed")
     post :submit_multiple_reboot, params: {:host_ids => host.id}, session: set_session_user(User.current)
     assert_redirected_to discovered_hosts_url
-    assert_match(/ERF50-4973/, flash[:error])
+    assert_equal "{:message=>\"Errors during reboot: request failed\", :host_details=>[{:name=>\"#{host.name}\", :error=>\"request failed\"}]}", flash[:error]
```

Comment:
The code is actually useful, I'd keep using it. Let me explain in different comment.